docs/Internal Systems/Meeting Minutes/23-3-22_FirstMeeting.md

```diff
@@ -0,0 +1,54 @@
+## Internal Systems Retrospective Minutes
+<br>
```

Comment:
**issue:** Could we avoid using HTML tag in markdown (ie. `<br>` tag)? Line spacing is controlled by the markdown renderer.